### ISSUE TITLE
Default filter settings are backward. 

### ISSUE THREAD

**Issue by jtheletter** [j0be/PowerDeleteSuite#10]
I ticked "filter by score", expecting it to ignore items scored above the given threshold. After running, I see it actually targeted those items and ignored the rest. 

Similarly, I ticked "filter by date", expecting it to ignore items older than the given duration. And after running, I see it actually targeted those and ignored the rest. 

I also expected the comments to be overwritten before deletion, as this is commonly known to  be required for security, and it's touted as a feature of this suite. However, I see after running that this wasn't enabled by default. 

I realize I should've read the exact language of these controls more closely. But wouldn't most users expect the defaults to be the opposite of their initial settings, as I did? Thanks for considering.

**Comment by schnerring** [comment]
> Similarly, I ticked "filter by date", expecting it to ignore items older than the given duration. And after running, I see it actually targeted those and ignored the rest.

Yeah, I wanted to delete everything older than am month, but now deleted everything newer than a month, while keeping the rest. This doesn't make any sense.

**Comment by curious-youna** [comment]
+1 on this, default should be `older than` option. The `newer than` seems weird.
